adagrams/game.py

```diff
@@ -1,13 +1,119 @@
 from random import randint
 
+LETTER_POOL = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+}
+
+LETTER_SCORE = {
+    "A": 1,
+    "E": 1,
+    "I": 1,
+    "O": 1,
+    "U": 1,
+    "L": 1,
+    "N": 1,
+    "R": 1,
+    "S": 1, 
+    "T": 1,
+    "D": 2, 
+    "G": 2,
+    "B": 3, 
+    "C": 3, 
+    "M": 3, 
+    "P": 3,
+    "F": 4, 
+    "H": 4, 
+    "V": 4, 
+    "W": 4, 
+    "Y": 4,
+    "K": 5,
+    "J": 8, 
+    "X": 8,
+    "Q": 10,
+    "Z": 10
+}
+
 def draw_letters():
-    pass
+    all_letters = []
+    hand = []
+
+    for letter, count in LETTER_POOL.items():
+        for _ in range(0, count):
+            all_letters.append(letter)
+
+    for _ in range(10):
```

Comment:
In Python, 10 on this line is what is known as a "Magic Number". We use the term to refer to hard coded values that just kind of appear without any context to them! If I wasn't familiar with Adagrams as either a user or someone working with the code, I wouldn't really know what that 10 meant! In these instances, it can be useful to create a constant either globally or locally that gives context to the number ten. As long as we picked a descriptive variable name, we'd be good! Something like the following would make sense:

```suggestion
    MAX_LETTERS_IN_HAND = 10
    for _ in range(MAX_LETTERS_IN_HAND):
```

Not only does this give us more context, but it would also give us some flexibility to use that particular context elsewhere if we needed to.